tests/unit/test_scanner.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from rat.scanner import *
+
+
+def test_scanner_invalid_scientific():
+    input_str = "g<lower=1.4ef> ~ normal(0.0, 1.0);"
+    parsed = Scanner(input_str)
+    with pytest.raises(TokenizeError):
```

Comment:
Since this is a custom Exception I don't think we need to add a match, but it still might be useful.